crates/sui-graphql-e2e-tests/tests/packages/friends.move

```diff
@@ -1,18 +1,18 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-//# init --addresses pkg=0x0 --simulator
+//# init --addresses P0=0x0 P1=0x0 --accounts A --simulator
```

Comment:
Mmm, can you have two addresses at the same location?

Reply:
You can and indeed for packages you must, because all packages must start at address 0x0, before they are published, and the act of publishing or upgrading them sets their address to the "correct" one.